docs/docs/monitoring/analytics/getting-started-with-analytics.mdx

```diff
@@ -213,6 +220,40 @@ analytics but lacks the necessary write performance to directly stream data to i
 
 :::
 
+### BigQuery
+
+These instructions assume you have running a PostgreSQL deployment in [CloudSQL](https://cloud.google.com/sql/docs/postgres).
+You can however use [Datastream for BigQuery](https://cloud.google.com/datastream/docs) to stream data from PostgreSQL
+hosted anywhere on any cloud.
+
+[Prerequisites](https://cloud.google.com/datastream/docs/before-you-begin) to using DataStream include:
+- creating a Google Cloud account where you create your Google Cloud project
+- enabling billing for the project
+- enabling the Datastream API
+- assigning Datastream Admin role to your user account
+- additional Datastream networking connectivity [requirements](https://cloud.google.com/datastream/docs/quickstart-replication-to-bigquery#requirements)
```

Comment:
I don't think it makes sense to list all of these prerequisites here. But linking to them is a good idea yeah